<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaprincipal</class>
 <widget class="QMainWindow" name="ventanaprincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contrarreloj</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTiempo">
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;center&gt;00:00:00&lt;/center&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="botonIniciar">
        <property name="text">
         <string>Iniciar/Parar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="botonCortar">
        <property name="text">
         <string>Cortar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="botonborrarultimo">
        <property name="text">
         <string>Borrar el último</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Inscritos:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLCDNumber" name="lcdtotalparcicipantes"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Han
llegado:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLCDNumber" name="lcdllegaron"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>&amp;Opciones</string>
    </property>
    <addaction name="actionRegistro"/>
    <addaction name="actionLlenar_numeros"/>
    <addaction name="actionResultados"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionAcerca_de_QT"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionAjustes_principales"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuOpciones"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAjustes_principales">
   <property name="text">
    <string>&amp;Ajustes principales</string>
   </property>
  </action>
  <action name="actionLlenar_numeros">
   <property name="text">
    <string>&amp;Llenar números</string>
   </property>
  </action>
  <action name="actionResultados">
   <property name="text">
    <string>R&amp;esultados</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>&amp;Acerca de</string>
   </property>
  </action>
  <action name="actionRegistro">
   <property name="text">
    <string>&amp;Registro</string>
   </property>
  </action>
  <action name="actionAcerca_de_QT">
   <property name="text">
    <string>A&amp;cerca de QT</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
